Application/Api/Billing/Bookkeeping/ApiBasket.php

```diff
@@ -341,11 +350,12 @@ public function getBasketTable($IsArchive = false)
 
     /**
      * @param string     $Identifier
+     * @param string     $Type
      * @param int|string $BasketId
      *
      * @return IFormInterface $Form
      */
-    public function formBasket($Identifier = '', $BasketId = '')
+    public function formBasket($Identifier = '', $Type = '', $BasketId = '')
```

Comment:
Codacy found an issue: [formBasket accesses the super-global variable $_POST.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)